parm/config/config.atmensanl

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+########## config.atmensanl ##########
+# configuration common to all atm ens analysis tasks
+
+echo "BEGIN: config.atmensanl"
+
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/lists/lgetkf_prototype.yaml
+export ATMENSYAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/lgetkf/lgetkf.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
```

Comment:
Thank you, @aerorahul , for noting this.  You are correct.  We no longer use `FV3JEDI_FIX` in UFS-DA ATM var and ens.   

`FV3JEDI_FIX` removed from `config.atmensanl` and `config.atmanl` at [30db276](https://github.com/RussTreadon-NOAA/global-workflow/commit/30db276ca20e8887ee686edb7722496508a63a42).   

`FV3JEDI_FIX` remains in `config.aeroanl` since I do not have a way to test UFS-DA Aerosol.